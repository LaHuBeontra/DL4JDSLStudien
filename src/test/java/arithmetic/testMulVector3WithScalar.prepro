function main() {
	import vec3 p1, scal numbers;

	vec3 result = p1 * numbers;

	export result;
}
