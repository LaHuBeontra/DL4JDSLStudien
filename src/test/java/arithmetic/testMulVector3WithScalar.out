DefaultPreProDataSet{
dataSetEntries=
[DataSetEntry{variableName='result', variableShape=[4, 3], startColumn=0, length=3}],
dataSet=
[[42.00,  0.00,  0.00],  
 [2.00,  0.00,  0.00],  
 [15.00,  0.00,  0.00],  
 [24.00,  0.00,  0.00]]

calculated_variables=
result=[[42.00,  0.00,  0.00],  
 [2.00,  0.00,  0.00],  
 [15.00,  0.00,  0.00],  
 [24.00,  0.00,  0.00]]
}